data/json/overmap/overmap_special/specials.json

```diff
@@ -7491,10 +7491,10 @@
       { "point": [ 3, 2, 1 ], "overmap": "fort_4c_1_north" }
     ],
     "locations": [ "wilderness" ],
-    "city_distance": [ 10, 100 ],
+    "city_distance": [ 0, 100 ],
```

Comment:
```suggestion
    "city_distance": [ 0, -1 ],
```

Reply:
What does having a max city distance of -1 do? Does that give it no max?